src/renderer/components/dock/edit-resource.store.ts

```diff
@@ -95,27 +95,24 @@ export class EditResourceStore extends DockTabStore<EditingResource> {
 export const editResourceStore = new EditResourceStore();
 
 export function editResourceTab(object: KubeObject, tabParams: Partial<IDockTab> = {}) {
-  // use existing tab if already opened
-  let tab = editResourceStore.getTabByResource(object);
+  const oldTab = editResourceStore.getTabByResource(object);
```

Comment:
How are these tab changes related to PR topic?

Reply:
They aren't, I thought I removed them when Roman noticed this too... sorry.

Reply:
This PR still has changes that are not related to PR topic.. If you really want these changes included then please add review comments why they are here.